src/main/java/thePackmaster/patches/warlockpack/NoFramePackPatch.java

```diff
@@ -0,0 +1,21 @@
+//package thePackmaster.patches.warlockpack;
+//
+//import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+//import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+//import com.evacipated.cardcrawl.modthespire.lib.SpireReturn;
+//import com.megacrit.cardcrawl.cards.AbstractCard;
+//import thePackmaster.packs.WarlockPack;
+//
+//@SpirePatch(
+//        clz = AbstractCard.class,
+//        method = "renderSkillPortrait"
+//)
+//
+//public class NoFramePackPatch {
```

Comment:
Should delete this file since it's all commented out. It can be recovered from history if needed -- or from this PR, for that matter :).